### PR TITLE
Automatically update built localized client files on src change

### PR DESCRIPTION
**Pull request by gideonthomas**

### CODE REVIEW

File: scripts/watch-client.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  error() {
+    console.log.apply(console, arguments ? log.args.call(arguments, "red") : null);
+  },
+  info() {
+    console.log.apply(console, arguments ? log.args.call(arguments, "green") : null);
+  }
+};
+
+function deleteFile(relativeFilePath) {
+  let deleteLocalizedFile = localeDirList.map(localeDir => fs.remove(path.join(localeDir, relativeFilePath)));
```
Comment (alicoding) [inline scripts/watch-client.js]: Why assign this now?

**Comment by alicoding** [comment]
r+ with nits

File: scripts/watch-client.js (hunk 1 above)
Comment (gideonthomas) [inline scripts/watch-client.js]: @cadecairos noted that it looked cleaner this way instead of stuffing all the code into one line.

Comment (alicoding) [inline scripts/watch-client.js]: Alright

**Pull request merged**